src/pytypes/onefuzztypes/models.py

```diff
@@ -799,6 +799,11 @@ class NetworkConfig(BaseModel):
     subnet: str = Field(default="10.0.0.0/16")
 
 
+class NetworkSecurityGroupConfig(BaseModel):
```

Comment:
Zooming out just a little bit: do you think we should name this something like `ProxyNetworkSecurityGroupConfig`?

Reply:
This is probably less important than the name of the field on the `InstanceConfig`.

Reply:
I think this is fine as-is, and we can even imagine it being re-used for something that isn't proxy-specific.